### DISCUSSION TITLE
Extra whitespace after links, how to remove?

### DISCUSSION THREAD

**Discussion by kristhayer11** [thegetty/quire#620] [Get Help] [answered]
Links in my markdown files generate an extra, blank whitespace after the link. 
Typically the extra whitespace appears just before punctuation. Like this:

<img width="300" alt="Screen Shot 2022-12-07 at 1 40 05 PM" src="https://user-images.githubusercontent.com/27781122/206279535-9fbd2bc6-09c8-4242-937e-c86fcd61fec5.png">

I think it has something to do with modifying the shortcode by adding something like {{- "" -}} at the end.

I've tried:
<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }}
target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>

But I can't get it to work. Has anyone solved this issue?

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @kristhayer11, I think your issue might be related to the same issue @mmpadilla15 had in https://github.com/thegetty/quire/discussions/478. The solution was to delete the empty second line. Let us know if that works!

**Reply by kristhayer11** [reply]
@Erin-Cecele Oh my gosh, such a simple fix! That solved the problem. Thank you, I never would have figured this out myself :-)
